### ISSUE TITLE
-re before -i

### ISSUE THREAD

**Issue by nathanaelneveux** [kkroening/ffmpeg-python#343]
Been through all the documentation and can't quite figure out how to reproduce a command like this:

ffmpeg -re -i inputfile.mp4 -codec copy -f flv rtmp://username:password@[wowza-address]:1935/live/myStream

the -re and flv pieces in particular

**Comment by Xiaozhuangzhuang** [comment]
just set "re=None" for input stream

**Comment by martimpassos** [comment]
so -n and -y options for overwriting work the same? like 
`stream = ffmpeg.input(image, y=None)`
?

**Comment by Xiaozhuangzhuang** [comment]
> so -n and -y options for overwriting work the same? like
> `stream = ffmpeg.input(image, y=None)`
> ?

no. you can `ffmpeg.input('in.mp4').output('out.mp4').run(overwrite_output=true)`

**Comment by sam97** [comment]
> no. you can ffmpeg.input('in.mp4').output('out.mp4').run(overwrite_output=true)

This works for `-y` but not for `-n`.

**Comment by shuZro** [comment]
was anyone able to figure this out?

**Comment by kaikiat** [comment]
> was anyone able to figure this out?
`ffmpeg.input(path ,re = None).output(RTMP_PATH, format='flv', codec = 'copy').run())`
